src/pages/editor/PublisherClaimsSettings.tsx

```diff
@@ -64,14 +64,15 @@ export const PublisherClaimsSettings = () => {
         {channelClaimsQuery.isPending ? (
           <PageSpinner aria-label={t('editor.institution.channel_claims.channel_claim')} />
         ) : channelClaimList && channelClaimList.length > 0 ? (
-          <ChannelClaimTable channelClaimList={channelClaimList} channelType={'publisher'} />
+          <ChannelClaimContext.Provider value={{ refetchClaimedChannels: channelClaimsQuery.refetch }}>
+            <ChannelClaimTable channelClaimList={channelClaimList} channelType={'publisher'} />
+          </ChannelClaimContext.Provider>
         ) : null}
       </TableContainer>
 
       <AddChannelClaimDialog
         open={openAddChannelClaimDialog}
         closeDialog={toggleAddChannelClaimDialog}
-        refetchClaimedChannels={channelClaimsQuery.refetch}
```

Comment:
Skjønner ikke helt hva som skjer / hvorfor dette virker 😅 
Her er propen tatt vekk, og i stedet bruker `AddChannelClaimDialog.tsx` context for refetch. Men `<ChannelClaimContext.Provider>` brukes bare over her, så da har vel ikke `AddChannelClaimDialog` tilgang til context der uansett? 🤔